client/src/components/metrics/Graph.tsx

```diff
@@ -76,7 +76,7 @@ export default function Graph() {
               style={{textAnchor: 'middle'}}
             />
           </XAxis>
-          <YAxis dataKey="students">
+          <YAxis dataKey="students" domain={[0, 'dataMax']}>
```

Comment:
the axis labels don't quite pop out in dark mode since default is black. not sure how to change with recharts but I imagine it'd be changing the style in Label below?